cpp/include/raft/lap/lap.cuh

```diff
@@ -60,7 +60,7 @@ class LinearAssignmentProblem {
 
  public:
   LinearAssignmentProblem(raft::handle_t const &handle, vertex_t size,
-                          vertex_t batchsize, weight_t epsilon)
+                          vertex_t batchsize, weight_t epsilon=1e-6)
```

Comment:
Note... this will provide a warning if weight_t is not float/double.  Cugraph builds a version where the weight_t is n int32_t.

There is a PR in the cugraph pipeline (https://github.com/rapidsai/cugraph/pull/1673) that will modify cugraph to specify this parameter in all cases.  Hopefully that will make this PR unnecessary.